app/models/profile.rb

```diff
@@ -25,12 +29,52 @@ class Profile < ApplicationRecord
 
   attr_reader :personal_key
 
+  aasm :fraud, column: :fraud_state, timestamps: true do
+    state :fraud_none, initial: true
+    state :fraud_reviewing, :fraud_rejected, :fraud_passed
+
+    event :fraud_review do
+      transitions from: :fraud_none, to: :fraud_reviewing
+    end
+
+    event :fraud_reject do
+      transitions from: [:fraud_none, :fraud_reviewing], to: :fraud_rejected
+    end
+
+    event :fraud_pass do
+      transitions(
+        from: [
+          :fraud_none,
```

Comment:
note: IMO we should not support the transition from `fraud_none` to `fraud_passed` without transitioning to `fraud_reviewing` in between. It is supported here to satisfy the current specs.